sass/elements/_html.scss

```
@@ -3,6 +3,7 @@ html {
     font-family: 'Fira Sans', sans-serif;
     font-size: calc-rem($size-body-mobile);
     background-color: $color-grey-900;
+    scroll-padding-top: 76px;
```

Then here just use the CSS var. And remove the `88px` definition for `tablet-landscape-up` (btw, you have a bug there as it would grow the scroll padding before the header height changes).

```suggestion
    scroll-padding-top: var(--scroll-padding-top);
```

---

> btw, you have a bug there as it would grow the scroll padding before the header height changes

I'm not sure I totally understand. Was this resolved in the latest commit though?

---

Yes! Latest changes look good. 👍 

About the "bug", I misread the code. You were using different breakpoints in `:root` and in `html` to change the `scroll-padding-top`; but since you were repeating the `scroll-padding-top` property probably there wasn't any issue. Anyway, with the CSS vars approach this is irrelevant.